packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -7354,6 +7365,31 @@ class Semantics extends SingleChildRenderObjectWidget {
   /// an [ExcludeSemantics] widget and then another [Semantics] widget.
   final bool excludeSemantics;
 
+  AttributedString? get _effectiveAttributedLabel {
+    return properties.attributedLabel ??
+          (properties.label == null ? null : AttributedString(properties.label!));
```

Comment:
Can we just create these in the constructor and pass them to the SemanticsProperties to avoid recreating them in every call to updateRenderObject?

Reply:
This will make Semantics non const. Is it possible to cache this value while keeping the constructor const?